boomer/boomer.go

```diff
@@ -127,6 +131,9 @@ func (b *Boomer) runWorker(n int) {
 		TLSHandshakeTimeout: time.Duration(b.Timeout) * time.Millisecond,
 		Proxy:               http.ProxyURL(b.ProxyAddr),
 	}
+	if b.H2 {
+		http2.ConfigureTransport(tr)
```

Comment:
I was able to read more about this and it seems easier. All we need to do is to set TLSNextProto to disable HTTP2: https://github.com/golang/go/blob/605e751b53a8bec7d1f51b2ccc0093e063358dc6/src/net/http/transport.go#L195

```
// to disable HTTP2
tr.TLSNextProto = make(map[string]func(string, *tls.Conn) http.RoundTripper)

// to enable HTTP2, tr.TLSNextProto must be nil.
```